### DISCUSSION TITLE
What is the use of require part in sqitch change

### DISCUSSION THREAD

**Discussion by wout-junius** [theory/sqitch#811] [Q&A]
So when creating a new change in sqitch you can add script that your new change is dependent on.
What effect does this give when your sqitch will execute the scripts in order of the sqitch.plan instead of parralell whit a dependency tree.

Is this here just for "documentation" or is there another reason/ use for this.

**Comment by theory** [comment]
You can require changes in other Sqitch projects! Here's a [2012 blog post](https://justatheory.com/2012/08/sqitch-depend-on-it/) about the feature. But otherwise yes, for documentation.
